app/components/Editor/components/Code.js

```diff
@@ -5,37 +5,44 @@ import type { SlateNodeProps } from '../types';
 import CopyButton from './CopyButton';
 import { color } from 'shared/styles/constants';
 
+function getCopyText(node) {
```

Comment:
Because `code` now contains individual `code-line` blocks we need to recombine them into a string with line breaks otherwise the copied string will all be on one line